packages/contracts/contracts/newsroom/NewsroomFactory.sol

```diff
@@ -26,6 +26,7 @@ contract NewsroomFactory is Factory {
   {
     address wallet = multisigFactory.create(initialOwners, initialRequired);
     newsroom = new Newsroom(name, charterUri, charterHash);
+    newsroom.addRole(msg.sender, "editor");
```

Comment:
only thing that's a little weird is passing in "editor" instead of using a constant or something

Reply:
the constant is set to private on the newsroom class.  but easy to do it the other way by changing it

Reply:
I'll change it if you think thats better

Reply:
or could add a function in Newsroom `addEditor` that uses its private constant there

Reply:
@nickreynolds updated